src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -215,6 +215,27 @@ class ActivityResourceIntTest : IntegrationTestBase() {
             assertThat(it).contains("PayPerSession").contains("""String "f": not one of the values accepted for Enum class: [H, F]""")
           }
       }
+
+      @Test
+      fun `Empty pay rates should still create activity and rules`() {
+        val missingPayRates = validJsonRequest(payRatesJson = """"payRates" : [],""")
+
+        val id = callCreateEndpoint(request = missingPayRates)
+
+        val courseActivity = repository.getActivity(id)
+        assertThat(courseActivity.payRates).isEmpty()
+        assertThat(courseActivity.courseScheduleRules.size).isGreaterThan(0)
+      }
+
+      @Test
+      fun `Missing pay rates should return bad request`() {
+        val missingPayRates = validJsonRequest(payRatesJson = "")
+
+        createActivityExpectingBadRequest(missingPayRates)
+          .expectBody().jsonPath("$.userMessage").value<String> {
+            assertThat(it).contains("payRates").contains("missing")
+          }
+      }
```

Comment:
We're well covered for tests demonstrating that we can handle updating an activity with missing pay rates but not for creating an activity with missing pay rates.